server/chat-commands/room-settings.ts

```diff
@@ -1232,7 +1233,7 @@ export const roomSettings: SettingsHandler[] = [
 	},
 	(room, user) => ({
 		label: "Modjoin",
-		permission: room.isPersonal ? user.can('editroom', null, room) : user.can('makeroom'),
+		permission: room.isPersonal ? 'editroom' : 'makeroom',
```

Comment:
This should probably still be `user.can('makeroom')`. Roomvoicing a Leader should not remove their ability to control rooms.